src/react/workflow/ExpirationForm.tsx

```diff
@@ -134,197 +182,185 @@ export function ExpirationForm({ bucketList, locations, prefix = '' }: Props) {
+                aria-describedby="error-prefix"
+                autoComplete="off"
+              />
+            }
+          />
+          <FormGroup
+            label="Tags"
+            id="filter.objectTags"
+            direction="horizontal"
+            error={errors[`${prefix}filter.objectTags`]?.message}
+            helpErrorPosition="bottom"
+            content={
+              <Controller
+                name={`${prefix}filter.objectTags`}
+                control={control}
+                defaultValue={[{ key: '', value: '' }]}
+                render={({ field: { onChange, value } }) => {
+                  return (
+                    <TagsFilter
+                      handleChange={onChange}
+                      control={control}
+                      fieldName={`${prefix}filter.objectTags`}
+                      tags={value}
+                      watch={watch}
                     />
-                  </T.ErrorContainer>
-                </T.Value>
-              </T.Row>
-              <T.Row
-                style={{
-                  maxHeight: 'initial',
-                  height: '100%',
-                  marginTop: '2rem',
-                  alignItems: 'baseline',
+                  );
                 }}
-              >
-                <T.Key>Tags</T.Key>
-                <T.Value>
-                  <Controller
-                    name={`${prefix}filter.objectTags`}
-                    control={control}
-                    defaultValue={[{ key: '', value: '' }]}
-                    render={({ field: { onChange, value } }) => {
-                      return (
-                        <TagsFilter
-                          handleChange={onChange}
-                          control={control}
-                          fieldName={`${prefix}filter.objectTags`}
-                          tags={value}
-                          watch={watch}
-                        />
-                      );
-                    }}
-                  />
-                </T.Value>
-              </T.Row>
-              <T.Row>
-                <T.ErrorContainer>
-                  <ErrorInput
-                    id="error-prefix"
-                    error={errors[`${prefix}filter.objectTags`]?.message}
-                  />
-                </T.ErrorContainer>
-              </T.Row>
-            </T.GroupContent>
-          </T.Group>
+              />
+            }
+          />
+        </FormSection>
 
-          <T.Group>
-            <T.GroupName>Action</T.GroupName>
-            <T.GroupContent>
-              <T.Row data-testid="toggle-action-expire-current-version">
-                <T.KeyTooltip
-                  tooltipMessage={`
+        <FormSection
+          title={{ name: 'Action' }}
+          forceLabelWidth={forceLabelWidth}
+        >
+          <T.GroupContent required>
+            <T.Row data-testid="toggle-action-expire-current-version">
+              <T.KeyTooltip
```

Comment:
I think we should try to get rid of using the legacy T style ;) 
And try to use `<FormGroup/>` for each of the Row/Action.